MapboxNavigation/InstructionsBannerViewLayout.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+extension InstructionsBannerView {
+    
+    func commonInit() {
+        setupViews()
+        setupLayout()
+        setupAvailableBounds()
+    }
+    
+    func setupViews() {
+        backgroundColor = .clear
+        
+        let maneuverView = ManeuverView()
+        maneuverView.backgroundColor = .clear
+        maneuverView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(maneuverView)
+        self.maneuverView = maneuverView
+        
+        let distanceLabel = DistanceLabel()
+        distanceLabel.translatesAutoresizingMaskIntoConstraints = false
+        distanceLabel.adjustsFontSizeToFitWidth = true
+        distanceLabel.minimumScaleFactor = 0.2
+        addSubview(distanceLabel)
+        self.distanceLabel = distanceLabel
+        
+        let primaryLabel = PrimaryLabel()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.adjustsFontSizeToFitWidth = true
+        primaryLabel.numberOfLines = 3
```

Comment:
This should be `1` if `secondaryLabel.text != nil`, `2` if `secondaryLabel.text == nil`